index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+    <header>
+        <span class="logo-placeholder">
+            Kenny Salazar
+        </span>
+        <div class="menu-icon">
+        </div>
+    </header>
+    <section class="headline">
+        <h1>Hey There.<br>I'm Kenny</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and 
+            have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="icons-container">
+            <span class="github-icon"></span><span class="linkedin-icon"></span><span class="fingers-icon"></span><span class="twitter-icon"></span>
+              <span class="m-icon"></span>
+        </div>
```

Comment:
When we are dealing with multiple links like in this instance, for better practices and semantics, we should group them in a `ul` tag and each list item inside the `ul` element should be as a `li` 😁 Also, please be sure to that each icon is wrapped with an `a` tag 🔥🤓